Jenkinsfile

```diff
@@ -4,7 +4,9 @@ pipeline {
   }
 
   agent {
-    label 'docker&&linux'
```

Comment:
Labels is important as we have linux  and windows machines with docker

Reply:
nevermind I got confused

Reply:
I mean thats an outdated change. I switched it to docker agent. Is that still going to be an issue? how can I change it to docker agent and linux?